packages/zoltan2/src/algorithms/partition/Zoltan2_AlgMultiJagged.hpp

```diff
@@ -3060,24 +3060,24 @@ void AlgMJ<mj_scalar_t, mj_lno_t, mj_gno_t, mj_part_t, mj_node_t>::
     }
   }
 
-  Kokkos::View<mj_part_t*, device_t> device_cumulative(
+  Kokkos::View<mj_scalar_t*, device_t> device_cumulative(
     "device_cumulative", num_cuts);
   typename decltype(device_cumulative)::HostMirror
```

Comment:
Consider for the future: If you're using `decltype` anyway, you might as well use `auto` here.

Reply:
Thanks. I did not ever consider that and it really cleans things up. I'll make those changes in a different PR.